### ISSUE TITLE
Theme possibly reusable for others

### ISSUE THREAD

**Issue by almereyda** [bencodezen/bencodezen#52]
The theme that powers the site https://www.bencodezen.io/ appears useful to third parties. Could it also live in a git submodule instead?

**Comment by bencodezen** [comment]
@almereyda Good point. I will use this issue to find a way to abstract it so other people can install it. Thank you for the suggestion!
